src/starknet_runner/runner.rs

```diff
@@ -257,107 +234,4 @@ mod test {
 
         assert_eq!(os_context, expected);
     }
-
-    #[test]
-    fn integration_test() {
```

Comment:
Is this test moved somewhere?

Reply:
Yes, to `tests/fibonacci.rs` along with `fibonacci.cairo`. This way it's a real integration test (without access to `pub(crate)` stuff).